source/module_io/input.cpp

```diff
@@ -641,6 +641,34 @@ void Input::Default(void)
     qo_thr = 1e-6;
     qo_screening_coeff = {};
 
+    //==========================================================
```

Comment:
I remember they have been added, I will check again

Reply:
Yes, they are already in the function.

Reply:
OK! But curious why the write_input.cpp file isn't listed in the left sidebar.

Reply:
It's at the bottom of the page, so it may be behind your taskbar. 
![image](https://github.com/deepmodeling/abacus-develop/assets/60290838/7018d759-e3be-42c5-b95d-e4457e1419e5)